### PR TITLE
fix(scripts): add eBPF jit warning

### PR DESCRIPTION
**Pull request by LucaGuerra**
Signed-off-by: Luca Guerra <luca@guerra.sh>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

> /kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

/area engine

> /area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

When using the eBPF probe we should double check if eBPF JIT is enabled. If not, the system could exhibit much poorer performance than expected. We did document this behavior in the past but it's also good to have a warning printed on the logs to make it easier to troubleshoot this kind of issues.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by FedeDP — APPROVED** [review]

**Review by leogr — COMMENTED** [review]
Hey @LucaGuerra 

We had a warning like this already implemented. IIRC, at some point, it was removed because we had issues like https://github.com/falcosecurity/falco/issues/1233

I believe it worth investigating a bit more, and making sure that reading from `/proc` works as expected under all circumstances (eg, https://falco.org/docs/getting-started/running/#docker).
Otherwise, we will risk giving misleading information to the users.

**Comment by LucaGuerra** [comment]
Thanks for the context @leogr ! I will investigate if we need additional checks on:
* whether the container has enough capability to read that value
* whether you need host network / host UTS namespace in order to do so

In that case, this warning should only be shown if the container is running with the correct privileges, and silent otherwise.

**Comment by japui** [comment]
correção (scripts): adicione aviso de jit eBPF #

File: scripts/falco-driver-loader [hunk 1]
```diff
@@ -471,6 +471,15 @@ load_bpf_probe() {
 	if [ -f "${HOME}/.falco/${BPF_PROBE_FILENAME}" ]; then
 		echo "* eBPF probe located in ${HOME}/.falco/${BPF_PROBE_FILENAME}"
 
+		if [ ! -f /proc/sys/net/core/bpf_jit_enable ] || [ $(cat /proc/sys/net/core/bpf_jit_enable) -eq 0 ]; then
```
Comment (FedeDP) [inline scripts/falco-driver-loader:474]: Perhaps we should use `HOST_ROOT/proc` here?

**Pull request closed without merging**